Jenkinsfile

```diff
@@ -0,0 +1,92 @@
+def initializeEnvironment() {
+
+    env.DRIVER_DISPLAY_NAME = 'Cassandra Ruby Driver'
+    env.DRIVER_METRIC_TYPE = 'oss'
+
+    env.GIT_SHA = "${env.GIT_COMMIT.take(7)}"
+    env.GITHUB_PROJECT_URL = "https://${GIT_URL.replaceFirst(/(git@|http:\/\/|https:\/\/)/, '').replace(':', '/').replace('.git', '')}"
+    env.GITHUB_BRANCH_URL = "${GITHUB_PROJECT_URL}/tree/${env.BRANCH_NAME}"
+    env.GITHUB_COMMIT_URL = "${GITHUB_PROJECT_URL}/commit/${env.GIT_COMMIT}"
+}
+
+def installDependencies() {}
```

Comment:
Nit: if these steps are empty, it's probably slightly better to get rid of them completely in the pipeline definition below, because they will appear in Blue Ocean as separate steps, which not only occupies visual space for nothing, but also incurs in a small time overhead as the executor goes from one step to the next one.

Reply:
Agreed, but at the moment these are placeholders.  I suspect we will have logic to place in here as the development of this build progresses, but at the moment I'm just trying to sketch out the overall process.

Reply:
These have been removed entirely in recent commits